course_discovery/apps/edly/api/v1/tests/test_views.py

```diff
@@ -0,0 +1,81 @@
+"""
+Unit tests for Edly API views.
+"""
+from rest_framework.reverse import reverse
+from course_discovery.apps.api.v1.tests.test_views.mixins import (
+    APITestCase,
+    SerializationMixin
+)
+from course_discovery.apps.core.tests.factories import (
+    USER_PASSWORD,
+    UserFactory
+)
+from course_discovery.apps.course_metadata.tests.factories import (
+    CourseRunFactory,
+    OrganizationFactory
+)
+from course_discovery.apps.course_metadata.models import (
+    Program
+)
+
+
+class ProgramViewSetTests(SerializationMixin, APITestCase):
+    program_list_url = reverse('edly:v1:program-list')
```

Comment:
You are adding class docstrings at some places but not at others. I think it would be better that u add class docstrings everywhere or nowhere. just a suggestion.